### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by AmineHLub**
## RoR Capstone

- Application : Wemdigo (short for where did my money go)

- Components: 
  - Splash Screen ( for sigining in/ signing up).
  - Navigation bar depending on where you are.
  - Hamburger menu (if logo is available): static and has no features it just opens and closes.
  - Category page: Shows list of user's category / can create new one.
  - Transaction page: shows all transactions belonging to a category / can add new one.

- Deployed : https://warm-fortress-20781.herokuapp.com/

- PS: it works but it's really buggy sometimes.

image

### CODE REVIEW

File: app/assets/stylesheets/categories.css [hunk 1]
```diff
@@ -0,0 +1,107 @@
+#categories {
+  padding: 10px 15px;
+  margin: 10px 2%;
+  border: 1px solid #434b54;
+  border-radius: 5px;
+}
+
+.description-container {
+  display: flex;
```
Comment (Lameck1) [inline app/assets/stylesheets/categories.css:9]: - [OPTIONAL] Let's strive to keep the code DRY. This property:
```CSS
display: flex
```
has been used several times in the CSS. Please refactor it to a class and apply the class in the HTML elements that share the same display property.

### Example code
```CSS
.d-flex {
  display: flex;
  align-items: center;
}
```

Then apply this class to HTML element that needs this behavior:

```HTML
  <section class="d-flex">
```

File: README.md [hunk 2]
```diff
@@ -1,58 +1,53 @@
...
-- Frameworks
-- Technologies used
+- Ruby
+- RoR
 
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+## Live Demo
+- [Click Me!](https://desolate-stream-21599.herokuapp.com/)
```
Comment (Lameck1) [inline README.md]: - [ ] One part of the project requirements expects that the project should be deployed and accessible online. It seems that this project was deployed but really not available. This is what we get when we access the live link:

  image
  
  Please try and work on the deployment and provide a working link.

File: README.md [hunk 3]
```diff
@@ -1,58 +1,53 @@
...
+## 🤝 Contributing
+- Microverse
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- [Gregoire Vella on Behance](https://www.behance.net/gregoirevella)
 
-## 🤝 Contributing
+  - [Her design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)
```
Comment (Lameck1) [inline README.md]: - [ ] The contributing section of the README should be a guide on how other developers can contribute to a project. It answers the question, how can one collaborate with you on this project?

  The details you've included should actually be in the `Acknowledgements` section. Please move them to the Acknowledgement section.
  
  Then, the details you have in the `Acknowledgements` section are just the general guidelines on how one should write the acknowledgments. Please replace them with the details you have here.
  
  Lastly, remember to re-write this `Contributing` section.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @AmineHLub, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Usage of partials to help keep code DRY ✔️
- Clean code ✔️
- Authentication with devise properly implemented ✔️
- The database schema reflects the provided structure ✔️
- The project design has been implemented ✔️
- Project setup with PostgreSQL ✔️
- Proper RSpec tests ✔️
- The MVP works as expected ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/assets/stylesheets/categories.css (hunk 1 above)
Comment (AmineHLub) [reply app/assets/stylesheets/categories.css:9]: Thank you for this. I've always been using it before this project, but I was really focusing on getting it to work and forgot to refactor I'll hopefully do that during the weekend and I'll make the burger menu functional too. Thanks a lot @Lameck1

File: README.md (hunk 2 above)
Comment (AmineHLub) [reply README.md]: I know right, It's really strange this is not the first time I deploy and we deployed last week too. I re-deployed again and the new link seems way better and faster and no bugs such as pictures not being loaded etc..

https://warm-fortress-20781.herokuapp.com/

I literally did the same thing again and now it's much better.

File: README.md (hunk 3 above)
Comment (AmineHLub) [reply README.md]: @Lameck1. I really thought I was putting this in the contribution section and I messed up. Thanks for rectifying this I fixed it

File: README.md (hunk 2 above)
Comment (peterrobert) [reply README.md]: This change has been implemented well

File: README.md (hunk 3 above)
Comment (peterrobert) [reply README.md]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @AmineHLub, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**